deploy/cloudbuild.yml

```diff
@@ -128,6 +146,7 @@ steps:
   - 'DOCKER_HOST=$_HOST:2376'
   - DOCKER_TLS_VERIFY=1
   - DOCKER_CERT_PATH=.
+  - 'BUILD_ID=$BUILD_ID'
```

Comment:
Is it just the ones with `$` references that we single-quote?

Reply:
Yep